schemas/v1.0/schema.yaml

```diff
@@ -0,0 +1,558 @@
+      context:
+        description: A runtime expression used to set the context for the condition to be applied on
+        type: string
+      condition:
+        description: The condition to apply
+        type: string
+    anyOf:
+      - type: object
+        properties:
+          type:
+            description: The type of condition to be applied
+            enum:
+              - simple
+              - regex
+              - jsonpath
+              - xpath
+            default: simple
+      - $ref: '#/$defs/criterion-expression-type-object'
+    required:
+      - condition
+    dependentRequired:
+      type:
+        - context
+    $ref: '#/$defs/specification-extensions'
+    unevaluatedProperties: false
+  criterion-expression-type-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#criterion-expression-type-object
+    description: An object used to describe the type and version of an expression used within a Criterion Object
+    type: object
+    properties:
+      type:
+        description: The type of condition to be applied
+        enum:
+          - jsonpath
+          - xpath
+      version:
+        description: A short hand string representing the version of the expression type
+        type: string
+    required:
+      - type
+      - version
+    allOf:
+      - if:
+          required:
+            - type
+          properties:
+            type:
+              const: jsonpath
+        then:
+          properties:
+            version:
+              const: draft-goessner-dispatch-jsonpath-00
+      - if:
+          required:
+            - type
+          properties:
+            type:
+              const: xpath
+        then:
+          properties:
+            version:
+              enum:
+                - xpath-10
+                - xpath-20
+                - xpath-30
+    $ref: '#/$defs/specification-extensions'
+  success-action-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#success-action-object
+    description: A single success action which describes an action to take upon success of a workflow step
+    type: object
+    properties:
+      name:
+        description: The name of the success action
+        type: string
+      type:
+        description: The type of action to take
+        enum:
+          - end
+          - goto
+      workflowId:
+        description: The workflowId referencing an existing workflow within the Arazzo description to transfer to upon success of the step
+        $dynamicRef: '#workflowId'
+      stepId:
+        description: The stepId to transfer to upon success of the step
+        $dynamicRef: '#stepId'
+      criteria:
+        description: A list of assertions to determine if this action SHALL be executed
+        type: array
+        uniqueItems: true
+        minItems: 1
+        items:
+          $ref: '#/$defs/criterion-object'
+    allOf:
+      - if:
+          properties:
+            type:
+              const: goto
```

Comment:
Should this `if` have `required: [goto]` ?

Reply:
should be `type`.  i'll add it

Reply:
er, yeah, I meant type. I see, that makes sense.